templates/s3_redundant/s3_redundant.py

```diff
@@ -181,6 +182,8 @@ def install(self):
             self.data['activeS3'] = active_s3.name
         active_s3.schedule_action('install').wait(die=True)
         self.logger.info('Installed s3 {}'.format(active_s3.name))
+        active_dmvm = self.api.service.get(template_uid=VM_TEMPLATE_UID, name=self.guid)
```

Comment:
the dm_vm's name is the guid of the `s3` not `s3_redundant`

Reply:
indeed thx.